itou/utils/htmx/utilities_for_testing.py

```diff
@@ -0,0 +1,32 @@
+"""
+Utilities useful when testing HTMX-related features.
+"""
```

Comment:
nit: pour l'instant on a de multiples `testing.py` dans le projet, qui sont effectivement les "utilities for testing"

Reply:
:ok: